### ISSUE TITLE
NVDA doesn't beep on Java Progress bars

### ISSUE THREAD

**Issue by thgcode** [nvaccess/nvda#13594]
### Steps to reproduce:
Open a Java app that contains a progress bar (for example I tested with this gist https://gist.github.com/brunosantanati/3c9e5778ba591b6218b6a210da1fc17f or Intellij).

### Actual behavior:
NVDA does not beep or speak the percentage when it finds a Java progress bar.

### Expected behavior:
NVDA should beep or speak on the progress bar, depending on the progress bar setting set.

#### NVDA installed/portable/running from source:
I tested with installed 2021.3.3 and source-master-c446688 and on both the bug occurs.

#### NVDA version:
Tested on 2021.3.3 and on source-master-c446688.

#### Windows version:
Tested with Windows 10 1903, but since it's a Java issue it might occur on previor or later versions of Windows.

#### Name and version of other software in use when reproducing the issue:
Tested with Java 11.0.14.1 (Intellij) and Java 17.0.2 (example gist).

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I tried both 2021.3.3 and source-master-c446688, it still occurs.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Since  it was a bug on the access bridge handling code, I did not run it, but I imagine it does.

**Comment by thgcode** [comment]
I will  send a pull request for this issue, I created the issue to send the pull request.
